### ISSUE TITLE
A11y Bug: Tooltip - Additional information isn't hoverable

### ISSUE THREAD

**Issue by schugartl** [ITS-HCD/nysds#974]
The [Tooltip component](https://designsystem.ny.gov/components/tooltip/) doesn't allow for mouse hover on the expanded tooltip information.

This interaction fails WCAG criterion [1.4.13 Content on Hover or Focus](https://www.w3.org/WAI/WCAG22/Understanding/content-on-hover-or-focus.html)

The current tooltip passes 2/3 of the requirement because it can be closed via Esc key (dismissible) and remains on screen until a user moves keyboard/hover focus (persistent). However it isn't hoverable with the mouse. Many low vision users with screen magnification software still utilize their mouse to navigate a page. Their magnified viewports can be fixed to the mouses location on the page. If they were to hover over something with this tooltip applied they may notice the new information pop up out of view and try to move their mouse over the new information. With the current design, these users are blocked because the tooltip information will disappear. 

My JS knowledge is very flimsy compared to HTML but from what I could find, the rough outline would be something like:
```
parent.onmouseover = function() {
  document.getElementById("popup").style.display = "block";
}

parent.onmouseout = function() {
  document.getElementById("popup").style.display = "none";
}
```
